includes/wikia/tasks/AsyncTaskList.class.php

```diff
@@ -284,12 +285,14 @@ protected function getExecutor() {
 	 * @throws AMQPExceptionInterface
 	 */
 	public function queue( AMQPChannel $channel = null, $priority = null ) {
-		global $wgUser;
-
 		$this->initializeWorkId();
 
 		if ( $this->createdBy == null ) {
-			$this->createdBy( $wgUser );
+			global $wgUser, $wgRequest;
```

Comment:
Relying on `RequestContext` here to get the `User` object would result in an infinite loop as the task is scheduled when the `User` object is initialized on the `RequestContext`. This cannot be remedied without refactoring `User` internals which is outside of the scope of this PR.